tests/Baseline/hilti.ast.basic-module/output

```diff
@@ -6,12 +6,16 @@
 
 #include <hilti/rt/libhilti.h>
 
-extern const char* __hlt_hlto_scope;
+extern uint64_t __hlt_hlto_scope;
```

Comment:
Could you add `#include <cstdint>` to `libhilti.h`? We already seem to pull it in implicitly, but would be good to be explicit.